edge/key.go

```diff
@@ -29,6 +30,11 @@ func (manager *Manager) SetKey(key string) error {
 		return err
 	}
 
+	u, _ := url.Parse(edgeKey.PortainerInstanceURL)
```

Comment:
Should we check error here?

Reply:
We could, but the front end doesn't allow you to add an endpoint/device without a scheme on the front of it.
Even if it's wrong, it'll error out later.